### PR TITLE
Project 1: Creating a data model

### PR DESCRIPTION
**Pull request by sarazaToussaint**
### In this Pull request I:

- Create a Blog app
- Initialize the project with Git.
- Make sure that the project has a Postgres database set up.
- Build project schema:
   - Create and run the necessary migration files.
   - Table and column names should match the ERD diagram
   - note: photo for users table should be a link to an image
   - Foreign keys should be included.
   - All columns that are foreign keys should have a corresponding index

### CODE REVIEW

File: db/migrate/20220818101111_create_posts.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :Title
+      t.text :Text
+      t.integer :CommentsCounter
+      t.integer :LikesCounter
+      t.references :users, null: false, foreign_key: true
```
Comment (alexander16108) [inline db/migrate/20220818101111_create_posts.rb]: hello there 👋🏾 , although you have done a fantastic job creating the table and populating it. there is still one tiny 👌🏾  detail left to address. 

you failed to reference the users table with its foreign key constraint `foreign_key : {to_table: another table}` .

File: db/migrate/20220818112851_create_comments.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :Text
+      t.references :users, null: false, foreign_key: true
+      t.references :posts, null: false, foreign_key: true
```
Comment (alexander16108) [inline db/migrate/20220818112851_create_comments.rb]: hello there 👋🏾 , although you have done a fantastic job creating the table and populating it. there is still one tiny 👌🏾  detail left to address. 

you failed to reference the users table with its foreign key constraint `foreign_key : {to_table: another table}` .

File: db/migrate/20220818113551_create_likes.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :users, null: false, foreign_key: true
+      t.references :posts, null: false, foreign_key: true
```
Comment (alexander16108) [inline db/migrate/20220818113551_create_likes.rb]: hello there 👋🏾 , although you have done a fantastic job creating the table and populating it. there is still one tiny 👌🏾  detail left to address. 

you failed to reference the users table with its foreign key constraint `foreign_key : {to_table: another table}` .

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @sarazaToussaint , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
 Although you did a very good job there is a tiny detail you need to work on before calling it a day with this task.



## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220818091943_create_users.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :Photo
+      t.text :Bio
+      t.integer :PostsCounter
+
```
Comment (elisha2kyakpo1) [inline db/migrate/20220818091943_create_users.rb:8]: - [x] [OPTIONAL] We can follow Ruby naming convention when we are writing variable or data attributes, `PostsCounter` should be `posts_counter`

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @sarazaToussaint, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- The PR title and description looks professional✔️
- No linter errors✔️
- Descriptive commit messages✔️
- You did well to make the changes requested✔️✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220818113551_create_likes.rb (hunk 3 above)
Comment (sarazaToussaint) [reply db/migrate/20220818113551_create_likes.rb]: Hi, @alexander16108 thank you for your review.

File: db/migrate/20220818091943_create_users.rb (hunk 4 above)
Comment (sarazaToussaint) [reply db/migrate/20220818091943_create_users.rb:8]: Hi @elisha2kyakpo1, thank you for approving my PR and mentioning this Ruby naming convention issue.

**Pull request merged**